packages/laconia-core/src/index.d.ts

```diff
@@ -23,6 +23,11 @@ declare namespace laconia {
       factory: LaconiaFactory | LaconiaFactory[],
       options?: FactoryOptions
     ): LaconiaHandler;
+    set(
```

Comment:
Would you mind adding a test under `laconia/packages/laconia-core/test/types/index.ts`? It's a single line that you should be adding.